ocs_ci/deployment/cnv.py

```diff
@@ -648,3 +648,135 @@ def disable_multicluster_engine(self):
             logger.error(f"Failed to disable multicluster engine\n{cmd_res.stderr}")
             return
         logger.info(cmd_res.stdout.decode("utf-8").splitlines())
+
+    def check_if_any_vm_instances(self, namespace=None):
+        """
+        Checks if any VMs or VM instances are running
+
+        Args:
+            namespace (str): namespace to check
+
+        Returns:
+            True if any VMs or VMi else False
+
+        """
+
+        vm_obj = OCP(kind=constants.VIRTUAL_MACHINE, namespace=namespace)
+        vmi_obj = OCP(kind=constants.VIRTUAL_MACHINE_INSTANCE, namespace=namespace)
+
+        return vm_obj.get(
+            out_yaml_format=False, all_namespaces=not namespace, dont_raise=True
```

Comment:
we can even set  the arg all_namespace to True so that it checks in all the namespaces for easy understanding

Reply:
with my logic, if namespace is not specified then it will be true for all namespaces, if specified then it will be false.